backend/app/controllers/projects.js

```diff
@@ -253,9 +255,14 @@ module.exports = {
         var thunkQuery = req.thunkQuery;
         co(function* () {
             yield * checkProjectData(req);
-            var updateObj = _.pick(req.body, ['title', 'description', 'startTime', 'closeTime', 'status', 'codeName']);
+            var updateObj = _.pick(req.body, ['title', 'description', 'startTime', 'closeTime', 'status', 'codeName', 'firstActivated']);
             var result = false;
             if (Object.keys(updateObj).length) {
+
+                // Update firstActivated if the status was changed from 0 to 1
+                if (updateObj.status == 1) {
```

Comment:
This should also check that `firstActivated` is currently `null`, otherwise this sets a new `firstActivated` value every time the `status` is set to `1`, not just the first time

Reply:
Good catch! Fixed!